app/models/user.rb

```diff
@@ -24,18 +26,10 @@ def password
     @password ||= Password.new(password_hash)
   end
 
-  def password=(new_password)
+  def reset_password(new_password)
```

Comment:
Why changed this? Wont it create issue in this place as we used `password==`. Not sure please let me know